src/main/java/step3/domain/RacingRule.java

```diff
@@ -0,0 +1,18 @@
+package step3.domain;
+
+import java.util.Random;
+
+public class RacingRule {
+
+  private static final Random random = new Random();
+  public static final int STANDARD_FOR_MOVING = 4;
```

Comment:
```suggestion
  private static final int STANDARD_FOR_MOVING = 4;
```
public으로 열어둘 필요는 없어보이네요